specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -13694,6 +13712,7 @@
         "state": {
           "type": "string",
           "title": "The current state of the compute node.",
+          "description": "Values are: idle - The node is not currently running a task; rebooting - The node is rebooting; reimaging - The node is reimaging; running - The node is running one or more tasks (other than a start task); unusable - The node cannot be used for task execution due to errors; creating - The Batch service has obtained the underlying virtual machine from Azure Compute, but it has not yet started to join the pool; starting - the Batch service is starting on the underlying virtual machine; waitingforstarttask - The start task has started running on the compute node, but waitForSuccess is set and the start task has not yet completed; starttaskfailed - The start task has failed on the compute node (and exhausted all retries), and waitForSuccess is set. The node is not usable for running tasks; unknown The Batch service has lost contact with the node, and does not know its true state; leavingpool - The node is leaving the pool, either because the user explicitly removed it or because the pool is resizing or autoscaling down; offline - The node is not currently running a task, and scheduling of new tasks to the node is disabled; preempted - The low-priority node has been preempted. Running tasks will be rescheduled when another node becomes available.",
```

Comment:
>Values [](start = 26, length = 6)

the order of these seems pretty random.  Might be good to try to organise them a bit more given the size of the enum (though I'm not sure whether there is an order that people would find predictable, other than plain alphabetical which may not be all that helpful)